src/app/components/universal/newSafetyDocumentUpdate/newSafetyDocumentUpdate.component.scss

```diff
@@ -0,0 +1,32 @@
+.container {
+
+  width: 80%;
+  height: 80%;
+
+}
+
+@media (max-width: 390px) {
+  .container {
+    width: 90%;
+    height: auto;
+    border: 8px dashed red;
```

Comment:
Atlas uses the rem unit for style sizing. I don't think anyone on the team has any clue how they work so just start putting random numbers in until it looks right. That's sort of a joke but you may need to do a little research on rem.

Reply:
I was just informed that a "rem" based off of the size of the root font size for the webpage. I am not 100% sure what ours is, but say it's 16px, then 1rem is equal to 16px. So you'd probably want 0.5rem here, but may need to adjust.

Reply:
Upon further investigation, it seems to be 16px. So you should be able to go off of that.

Reply:
thank you for the info on rem, I didn't know about it since I am so used to pixel but apparently it's used extensively for responsive web design. I shall adjust fire accordingly.